src/css/dark.css

```diff
@@ -110,6 +109,7 @@ input.number:focus {
 		}
 
 		#menubar .menu .options {
+      cursor: pointer;
```

Comment:
yeah, i'll just leave it for now since i've got a bunch of other changes in that PR related to changes in the CSS.